tests/api/v1/collectorGroups/get.js

```diff
@@ -0,0 +1,189 @@
+    .then((cg) => {
+      collectorGrp = cg;
+      done();
+    })
+    .catch(done);
+  });
+
+  afterEach(u.forceDelete);
+  after(tu.forceDeleteUser);
+
+  it('get by name ok', (done) => {
+    api.get(path + '/' + collectorGrpObj.name)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.OK)
+      .expect((res) => {
+        expect(res.body.id).to.equal(collectorGrp.id);
+      })
+      .end(done);
+  });
+
+  it('get by id ok', (done) => {
+    api.get(path + '/' + collectorGrp.id)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.OK)
+      .expect((res) => {
+        expect(res.body.id).to.equal(collectorGrp.id);
+      })
+      .end(done);
+  });
+
+  it('find ok', (done) => {
+    api.get(path)
+      .set('Authorization', token)
+      .expect(constants.httpStatus.OK)
+      .expect((res) => {
+        expect(res.body.length).to.equal(1);
+        expect(res.body[0].name).to.equal('___-CollGroup');
+      })
+      .end(done);
+  });
+
+  describe('with collectors > ', () => {
+    let coll1;
+    let coll2;
+    beforeEach(() => {
+      const collector1 = u.getCollectorToCreate();
+      collector1.name += '1';
+      return Collector.create(collector1)
+        .then((c1) => {
+          coll1 = c1;
+          const collector2 = u.getCollectorToCreate();
+          collector2.name += '2';
+          return Collector.create(collector2);
+        })
+        .then((c2) => {
+          coll2 = c2;
+          return collectorGrp.addCollectors([coll1, c2]);
+        });
+    });
+
+    it('get by name, returns collectors array', (done) => {
+      api.get(path + '/' + collectorGrpObj.name)
+        .set('Authorization', token)
+        .expect(constants.httpStatus.OK)
+        .expect((res) => {
+          expect(res.body.id).to.equal(collectorGrp.id);
+          expect(res.body.collectors.length).to.equal(2);
+
+          /*
+            Response has collector id, name, status and lastHeartbeat
+            Returned collector list is not in particular order
+           */
+          const respColl = res.body.collectors;
+          const collNamesInResp = [respColl[0].name, respColl[1].name];
+          expect(collNamesInResp).to.have.members([coll1.name, coll2.name]);
+
+          const collStatusInResp = [respColl[0].status, respColl[1].status];
+          expect(collStatusInResp).to.have
+            .members([coll1.status, coll2.status]);
+        })
+        .end(done);
+    });
+
+    it('get by id, returns collectors array', (done) => {
+      api.get(path + '/' + collectorGrp.id)
+        .set('Authorization', token)
+        .expect(constants.httpStatus.OK)
+        .expect((res) => {
+          expect(res.body.id).to.equal(collectorGrp.id);
+          expect(res.body.collectors.length).to.equal(2);
+
+          /*
+            Response has collector id, name, status and lastHeartbeat
+            Returned collector list is not in particular order
```

Comment:
So, I expected the returned collector list to be ordered by name (we already have order: ['name'] in the included collector scope). But for some reason, its not always ordered. I tried to figure this out for some time, but could not maintain the order. Any ideas? Maybe another sequelize problem :(